src/main/resources/static/js/reservation-legacy.js

```
@@ -2,145 +2,145 @@ let isEditing = false;
+    input.type = type;
+    input.className = 'form-control';
+    return input;
 }
 
 function createActionButton(label, className, eventListener) {
-  const button = document.createElement('button');
-  button.textContent = label;
-  button.classList.add('btn', className, 'mr-2');
-  button.addEventListener('click', eventListener);
-  return button;
+    const button = document.createElement('button');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createActionButton'. Consider to delete it or rename it.

'createActionButton' is defined here:


(lint/suspicious/noRedeclare)